store/store.go

```diff
@@ -282,6 +282,10 @@ func (bs *BlockStore) LoadSeenCommit(height int64) *types.Commit {
 	return commit
 }
 
+func (bs *BlockStore) Close() error {
```

Comment:
godoc is missing. This also seems like a separate feature to this PR. I would consider splitting this up to a separate PR and looking to add this to the state store (possibly tx indexer and evidence pool) as well

Reply:
Will add a godoc comment. This is required so that I can call a constructor more than once in a test. Without it, you can't acquire a new connection to the data bases. It may make sense to split into a different PR and make sure it is also tested more thoroughly. Any idea how I can multiple DB connections in a test for now?

Reply:
Is it possible to just use the in memory db? -> `dbStore := New(dbm.NewMemDB())`

Reply:
Didn't realize this existed! Will update.

Reply:
Agreed, I see that a _lot_ of pre-existing code is missing meaningful godoc. I thought at first that was just because it was generated code on that basis. Strong 👍🏼 for tracking a separate issue to improve on that.